src/classes/FreshbooksAPI.cls

```diff
@@ -129,7 +129,15 @@ public with sharing class FreshbooksAPI {
 		//client.addChildElement('language',null,null).addTextNode(language);
 		//client.addChildElement('currency_code',null,null).addTextNode(currencyCode);
 		appendTagAndText(client,'notes',a.Freshbooks_Notes__c);
-		appendTagAndText(client,'p_street1',a.BillingStreet);
+		
+                // If we've got two lines of street address, put them into separate lines, if not put into one and clean it up.
+                String[] str = a.BillingStreet.split('\r\n');
```

Comment:
Not really, if I use Windows and modify the address to be more then one line, the object will contain '\r\n'. Tested that on linux machine with the same result - '\r\n'. Please test on mac by:
- modify the GOof account in my org, so that the address is one line.
- modify the GOof account in my org, so that the address is multi line.
- run following code in dev console and check the debug messages:
```java
Account acc = [select id, BillingStreet, Name, Phone, Fax
               from Account where id = '001K0000019vsim'];
FreshbooksAPI.saveClient(acc);
```
This is what I got on linux (meaning the split by \r\n worked well) having the following as BillingStreet:
```
this is good,
skdjfsldkf,
sdfds
```
Check the p_street1:
```xml
requestDoc: <?xml version="1.0" encoding="UTF-8"?><request method="client.create"><client><organization>GOof</organization><work_phone></work_phone><fax></fax><p_street1>this is good, skdjfsldkf, sdfds</p_street1></client></request>
```